theano/tests/test_printing.py

```diff
@@ -270,6 +271,26 @@ def test_debugprint():
 
     assert s == reference
 
+    # test clients
+    s = StringIO()
+    f = theano.function([A, B, D], [A + B, A + B - D])
+    debugprint(f, file=s)
+    s = s.getvalue()
+    # The additional white space are needed!
+    reference = '\n'.join([
+"Elemwise{add,no_inplace} [id A] ''   0 clients:[('output', ''), ('[id C]', 1)]",
+" |A [id D]",
+" |B [id E]",
+"Elemwise{sub,no_inplace} [id C] ''   1",
+" |Elemwise{add,no_inplace} [id A] ''   0 clients:[('output', ''), ('[id C]', 1)]",
+" |D [id F]",
+    ]) + '\n'
```

Comment:
Right. I don't want to change the default, so I updated this PR to make this an option.